app/models/similar_occupation_standards.rb

```diff
@@ -29,37 +33,56 @@ def similar_to
   private
 
   def query
-    {
-      size: RESULTS_SIZE,
-      min_score: MINIMUM_SCORE,
-      query: {
-        bool: {
-          should: [
-            {match: {
-              title: {query: occupation_standard.title, boost: 5}
-            }},
-            {match: {
-              work_process_titles: {
-                query: occupation_standard.work_processes.pluck(:title).to_sentence
-              }
-            }},
-            {match: {
-              ojt_type: {query: occupation_standard.ojt_type, boost: 0.5}
-            }},
-            {match: {
-              state: {query: occupation_standard.registration_agency&.state&.abbreviation}
-            }}
-          ],
-          minimum_should_match: 1,
-          must_not: [
-            {
-              term: {
-                _id: occupation_standard.id
-              }
-            }
-          ]
+    definition = Search.new
```

Comment:
@littleforest I noticed that there's a more DSL way to do this instead of the imperative I'm doing. Not sure which one you had in mind